flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -269,26 +269,26 @@
 	 * Percentage of heap space to remove from containers (YARN / Mesos), to compensate
 	 * for other JVM memory usage.
 	 */
-	public static final String CONTAINERED_HEAP_CUTOFF_RATIO = "containered.heap-cutoff-ratio";
+	public static final String CONTAINER_HEAP_CUTOFF_RATIO = "container.heap-cutoff-ratio";
```

Comment:
IMHO the `containered` prefix reflects the meaning of this config key better than `container`. It cuts off a fraction from the containerized Flink process, not from the container itself which the latter would imply.